README.md

```diff
@@ -117,9 +120,9 @@ docker build -t lambda-php .
 
 The example PHP functions are also built into the example image, so they can be run with Docker:
 ```
-docker run lambda-php hello '{"name": "World"}'
+docker run lambda-php functions.hello '{"name": "World"}'
```

Comment:
Looks like changes from your other PR are getting into this, can you separate into different branches?

Reply:
Resolved